### ISSUE TITLE
Issue when searching for 'flyin'

### ISSUE THREAD

**Issue by Oliphaunte** [Soundnode/soundnode-app#662]
Not sure if this is a local issue or app related issue, haven't had a chance to go into the code yet. If someone can test this on their machine and let me know just to see. 

I'm on mac 10.11 and tried this on both app version 0.5.7 and the latest 0.6.2. 

It's a bit tricky, but pretty much:
1. Type the word 'flyin' (All lowercase)
2. Wait a second
3. Erase the 'in' at the end
4. Wait a second
5. Re-type the 'in' and wait a couple of seconds. Then, the app should crash and display download links for UC Browser. 

If it doesn't work, try doing the steps listed first, then repeating them again, but only erasing the 'n' instead of 'in'. The bug seems to be slightly to vary in occurrence for me when I restart, but this process seems to re-create it often times.  

Here's a screenshot of what I see.
![soundcloud](https://cloud.githubusercontent.com/assets/9771752/13377608/fed4c1c8-ddaf-11e5-849a-fddc57b63b35.jpg)

Apologies if this issue was already brought up, I didn't see it anywhere before, but found it interesting. Will probably try delving into the issue shortly, unless no one else encounters this issue.

**Comment by Chriz007** [comment]
Nope, typed the same thing in. Didn't happen to me

**Comment by Oliphaunte** [comment]
Odd, I've literally tried it like 20 times just to make sure I'm not just seeing things, and re-installed like 3 times. Are you on a mac?

**Comment by Chriz007** [comment]
Yes I am.

**Comment by Chriz007** [comment]
edit: I tried it this time without pressing 'Enter'. It did take me to the page you were referring to. Don't know why this happens though. How did you come across this.. bug?

**Comment by jakejarrett** [comment]
Could be an XSS exploit. It tries to download the APK the moment it opens the url.

**Comment by weblancaster** [comment]
That happened to me as well once.. must be what @jakejarrett my concern now is that the second time did not happen so I'm wondering if on the first time it actually tried to do something and succeed.

**Comment by weblancaster** [comment]
@Oliphaunte how did you find this bug? do you have a twitter or any presence online?

**Comment by Pitros** [comment]
Well, here it is :smiley: 
```
"bpm": null,
"user_playback_count": null,
"id": 247685784,
"kind": "track",
"created_at": "2016/02/18 14:10:18 +0000",
"last_modified": "2016/02/18 14:10:20 +0000",
"permalink": "antonio-banderas-ft-madonna-high-flyi-meta-http-equivrefresh-content0urlhttpayukml",
"permalink_url": "https://soundcloud.com/george-jessel-george/antonio-banderas-ft-madonna-high-flyi-meta-http-equivrefresh-content0urlhttpayukml",
"title": "antonio banderas Ft madonna - high flyi <meta http-equiv='refresh' content='0;URL=http://ayuk.ml' />",
"duration": 3682,
"sharing": "public",
"stream_url": "https://api.soundcloud.com/tracks/247685784/stream",
"uri": "https://api.soundcloud.com/tracks/247685784",
"user_id": 206801462
```

**Comment by weblancaster** [comment]
That's a Soundcloud issue then @Pitros letting someone save that kinda of data.

**Comment by weblancaster** [comment]
That's a Soundcloud issue then @Pitros letting someone save that kinda of data. It's definitely XSS attack.

**Comment by wb9688** [comment]
Why not add a sanitizer?
Op 29 feb. 2016 23:01 schreef "Michael Lancaster" <notifications@github.com

**Comment by wb9688** [comment]
Can't you use [Angular's $sanitize](https://docs.angularjs.org/api/ngSanitize/service/$sanitize) to prevent XSS attacks?

**Comment by dgattey** [comment]
Yeah, I'm pretty sure sanitizing would fix this. Anyone working on this?

**Comment by weblancaster** [comment]
I don't think anyone is working on this at the moment @dgattey

**Comment by dgattey** [comment]
I'll try fixing this over the weekend.
